### PR TITLE
Fix: Move images in zoomable light table

### PR DESCRIPTION
**Pull request by monsieurmona**
* Remove drag and drop registration for zoomable light table
* Pan if custom sorting is not required

### CODE REVIEW

File: src/views/lighttable.c [hunk 1]
```diff
@@ -2501,29 +2503,51 @@ void gui_init(dt_view_t *self)
...
+    // only in file manager
+    // only in light table
+    // only if custom image order is selected
+    dt_view_t *current_view = darktable.view_manager->current_view;
+    if (layout == file_manager_layout &&
+        darktable.collection->params.sort == DT_COLLECTION_SORT_CUSTOM_ORDER &&
+        current_view &&
+        current_view->view(self) == DT_VIEW_LIGHTTABLE)
+    {
+      custom_image_order_required = TRUE;
```
Comment (TurboGit) [inline src/views/lighttable.c]: return TRUE;

File: src/views/lighttable.c [hunk 2]
```diff
@@ -2501,29 +2503,51 @@ void gui_init(dt_view_t *self)
...
+    }
+  }
+
+  return custom_image_order_required;
```
Comment (TurboGit) [inline src/views/lighttable.c]: return FALSE.

File: src/views/lighttable.c [hunk 3]
```diff
@@ -2501,29 +2503,51 @@ void gui_init(dt_view_t *self)
   g_signal_connect(G_OBJECT(display_profile), "value-changed", G_CALLBACK(display_profile_callback), NULL);
 }
 
+static gboolean _is_custom_image_order_required(dt_view_t *self)
+{
+  gboolean custom_image_order_required = FALSE;
```
Comment (TurboGit) [inline src/views/lighttable.c]: can be removed with proposed change below

File: src/views/lighttable.c (hunk 2 above)
Comment (monsieurmona) [reply src/views/lighttable.c]: Done

File: src/views/lighttable.c (hunk 3 above)
Comment (monsieurmona) [reply src/views/lighttable.c]: Done

File: src/views/lighttable.c (hunk 1 above)
Comment (monsieurmona) [reply src/views/lighttable.c]: Done

File: src/views/lighttable.c (hunk 3 above)
Comment (TurboGit) [reply src/views/lighttable.c]: Perfect! Thanks.

**Comment by monsieurmona** [comment]
Thanks for your help!

**Pull request merged**